mne/viz/decoding.py

```diff
@@ -11,6 +11,127 @@
 import numpy as np
 import warnings
 
+def plot_topopatterns(csp, layout=None, vmin=None, vmax=None, sensors=True,
```

Comment:
actually, I think we would move plot_patterns to viz.decoding, since it would also be called by classes based on linear models (LinearClassifier & LinearRegressor, see #2290)
There will be some CSP.plot_patterns, LinearClassifier.plot_patterns or LinearRegressor.plot_pattern but it would just call the viz.decoding function (e.g. using CSP1, CSP2, CSP3 instead of time in ms)